user/enterprise/ssl-certificate-management.md

```diff
@@ -88,131 +88,167 @@ Upon restart you can then verify whether SSL verification is now working. If you
 
 ## Using a Let's Encrypt SSL Certificate
 
-Travis CI Enterprise works well together with a Let's Encrypt SSL Certificate. To obtain one for your domain, please follow the steps below.
+You can use a certificate from [Let's Encrypt](https://letsencrypt.org/) instead of a self-signed certificate or a certificate purchased from a trusted certificate authority. Certificates from Let's Encrypt are free and behave the same as those purchased from a trusted certificate authority. More information about Let's Encrypt can be found [here](https://letsencrypt.org/about/).
 
-What you need:
+What you will need:
 
-1. An email address Let's Encrypt can send emails to. This will be used to notify about urgent renewal and security notices.
+1. An email address (Let's Encrypt will send notifications regarding urgent renewal and security issues).
 2. A domain name under which your installation is available (we're using `travis.example.com` in this guide).
 
-**Please note: This change will cause downtime for your system,for this reason, we recommend to perform this process in a maintenance window.**
+> The following steps require downtime for your Travis CI Enterprise instance. For this reason we recommend that you perform these steps during a maintenance window.
+
+### Installing certbot
+
+We will be using [certbot](https://certbot.eff.org/#ubuntutrusty-other) to obtain a SSL certificate from Let's Encrypt. To install certbot:
+
+1. Open a ssh connection to the platform machine.
+2. Add the certbot PPA:
```

Comment:
Might be worth expanding PPA for folks who aren't familiar with Ubuntu

Reply:
Addressed in 192be263